src/audio/base_fw.c

```diff
@@ -323,6 +325,69 @@ static uint32_t basefw_get_ext_system_time(uint32_t *data_offset, char *data)
 	return IPC4_UNAVAILABLE;
 }
 
+static int basefw_register_kcps(bool first_block,
+				bool last_block,
+				uint32_t data_offset_or_size,
+				const char *data)
+{
+	if (!(first_block && last_block))
+		return IPC4_INVALID_REQUEST;
+
+	if (core_kcps_adjust(0, *(int *)data))
+		return IPC4_FAILURE;
+
+	return IPC4_SUCCESS;
+}
+
+static int basefw_KCPS_allocation_request(struct ipc4_resource_kcps *request)
```

Comment:
This "\_KCPS\_" just looks weird.

Reply:
agree, this is due to people know mcps more, maybe add a comments is better.